libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/baseChart/PluggableBaseChart.tsx

```diff
@@ -220,7 +220,7 @@ export class PluggableBaseChart extends AbstractPluggableVisualization {
         insight: IInsightDefinition,
         executionFactory: IExecutionFactory,
     ): void {
-        const { dimensions = { height: undefined }, custom = {}, locale } = options;
+        const { dimensions = { height: undefined }, custom = {}, locale, dateFormat } = options;
```

Comment:
Hello @no23reason ,
We have FF responsiveUiDateFormat  to config the date format with priority like: userConfig -> project config -> domain config. See the FF detail [here](https://github.com/gooddata/puppet/blob/develop/modules/c4/files/c4_platform_settings_catalog.json#L2193)

So I would read the dateFormat from the Settings for current user. So I would like to change like [this](https://github.com/lanhq/gooddata-ui-sdk/commit/fd555a2627b532543a0453f19c1deab0bd99a201). Then From AD, KD we just need send the feature flags setting to PluggableVisualization and no need to make a new property `dateFormat` in `IVisProps`.

What do you think?

Reply:
The change makes sense I think. However wouldn't it be easier to keep the `IVisProps.dateFormat` and fill it in AD/KD?

Reply:
So for `InsightView`, after load Settings for current user then read `responsiveUiDateFormat` and convert to `dateFormat` in `IVisProps`. Similar for AD,KD we still send `dateFormat` to SDK. 
This way is more clear right?

Reply:
Yes, that makes sense to me.